Gemfile.lock

```diff
@@ -1,57 +1,57 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (5.2.1)
-      actionview (= 5.2.1)
-      activesupport (= 5.2.1)
+    actionpack (5.2.2.1)
+      actionview (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       rack (~> 2.0)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.2)
-    actionview (5.2.1)
-      activesupport (= 5.2.1)
+    actionview (5.2.2.1)
+      activesupport (= 5.2.2.1)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.0.3)
-    activemodel (5.2.1)
-      activesupport (= 5.2.1)
-    activerecord (5.2.1)
-      activemodel (= 5.2.1)
-      activesupport (= 5.2.1)
+    activemodel (5.2.2.1)
+      activesupport (= 5.2.2.1)
+    activerecord (5.2.2.1)
+      activemodel (= 5.2.2.1)
+      activesupport (= 5.2.2.1)
       arel (>= 9.0)
-    activesupport (5.2.1)
+    activesupport (5.2.2.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
     arel (9.0.0)
     builder (3.2.3)
-    concurrent-ruby (1.1.1)
+    concurrent-ruby (1.1.5)
     crass (1.0.4)
     double_entry (1.0.1)
       activerecord (>= 3.2.0)
       activesupport (>= 3.2.0)
       money (>= 6.0.0)
       railties (>= 3.2.0)
-    erubi (1.7.1)
-    i18n (1.1.1)
+    erubi (1.8.0)
+    i18n (1.6.0)
       concurrent-ruby (~> 1.0)
-    json (2.1.0)
-    jwt (1.5.6)
-    jwt_signed_request (2.4.0)
-      jwt (~> 1.5.0)
+    json (2.2.0)
+    jwt (2.1.0)
+    jwt_signed_request (2.5.1)
+      jwt (>= 1.5.0, < 2.2.0)
       rack
     loofah (2.2.3)
       crass (~> 1.0.2)
       nokogiri (>= 1.5.9)
-    method_source (0.9.1)
-    mini_portile2 (2.3.0)
+    method_source (0.9.2)
+    mini_portile2 (2.4.0)
```

Comment:
### [mini_portile2](http://github.com/flavorjones/mini_portile)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.3.0 → 2.4.0

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **10** commits. See the full changes on [the compare page](https://github.com/flavorjones/mini_portile/compare/v2.3.0...v2.4.0).

These are the individual commits:
- (4fef7ad) [convert to using windows-ruby-dev-tools-release](https://github.com/flavorjones/mini_portile/commit/4fef7ada7096aee14c5fcdaddd2292c52d84d4e7)
- (f0f6b7d) [concourse: test most-recent two rubies](https://github.com/flavorjones/mini_portile/commit/f0f6b7d447e8516cb6da649b7c80a1bca43bb330)
- (82af422) [update test:examples to libiconv 1.15](https://github.com/flavorjones/mini_portile/commit/82af4225a7ab5711981fd3b8df4b17aeb51f851c)
- (a9fd589) [Skip progress report when Content-Length is unavailable](https://github.com/flavorjones/mini_portile/commit/a9fd5895718b4003629f7e0d2715834ae05cbe98)
- (ede15b9) [Make version in changelog fit release version.](https://github.com/flavorjones/mini_portile/commit/ede15b950100b886d95d0e2be55cdd8a697381c5)
- (51f5ac4) [Merge pull request #87 from halfbyte/patch-1](https://github.com/flavorjones/mini_portile/commit/51f5ac46d4082432bbb9d13aa9eb7e2737520e90)
- (ce0e270) [Merge pull request #86 from eagletmt/skip-progress-when-chun…](https://github.com/flavorjones/mini_portile/commit/ce0e270662917e333e7d25cee682c3ad9fe815f0)
- (9e699c6) [update dev dependencies](https://github.com/flavorjones/mini_portile/commit/9e699c64e8b749e67865b14ace9aa398bfd4839f)
- (2499fcb) [update CHANGELOG in preparation for v2.4.0](https://github.com/flavorjones/mini_portile/commit/2499fcba523f77162acb5c51711c77988aa1c96a)
- (cf441e5) [version bump to v2.4.0](https://github.com/flavorjones/mini_portile/commit/cf441e5a007e1721ae7d67a69a920ea761f903fe)


</details>